jobs/build/ocp4/build.groovy

```diff
@@ -87,6 +88,37 @@ def initialize() {
     return planBuilds()
 }
 
+def setBuildType() {
+    if (!isTriggeredByOrganicLifeForms()) {
+        echo "Assuming *PROD* build, triggered by automation"
+        scratch = false
+        return
+    }
+    askBuildType()
```

Comment:
should not do this if hooman checked the `SCRATCH` option; only "are you sure" on prod builds.